test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/IncludeSearchTestFixture.cs

```diff
@@ -91,27 +115,156 @@ Observation CreateObservation(Patient patient, CodeableConcept code)
                         Status = ObservationStatus.Final,
                         Code = code,
                         Subject = new ResourceReference($"Patient/{patient.Id}"),
-                        Performer = new List<ResourceReference>
+                        Performer = new List<ResourceReference>()
+                        {
+                            new ResourceReference($"Organization/{organization.Id}"),
+                            new ResourceReference($"Practitioner/{practitioner.Id}"),
+                        },
+                    }).Result.Resource;
+            }
+
+            Patient CreatePatient(string familyName, Practitioner practitioner, Organization organization)
+            {
+                return TestFhirClient.CreateAsync(
+                    new Patient
+                    {
+                        Meta = meta,
+                        Name = new List<HumanName> { new HumanName { Family = familyName } },
+                        GeneralPractitioner = new List<ResourceReference>()
+                        {
+                            new ResourceReference($"Practitioner/{practitioner.Id}"),
+                        },
+                        ManagingOrganization = new ResourceReference($"Organization/{organization.Id}"),
+                    }).Result.Resource;
+            }
+
+            MedicationDispense CreateMedicationDispense(MedicationRequest medicationRequest, Patient patient, Medication medication)
+            {
+               return TestFhirClient.CreateAsync(
+                    new MedicationDispense
+                    {
+                        Meta = meta,
+                        AuthorizingPrescription = medicationRequest == null ? null : new List<ResourceReference>
+                        {
+                            new ResourceReference($"MedicationRequest/{medicationRequest.Id}"),
+                        },
+                        Subject = new ResourceReference($"Patient/{patient.Id}"),
+                        Performer = new List<MedicationDispense.PerformerComponent>()
                         {
-                            new ResourceReference($"Organization/{Organization.Id}"),
-                            new ResourceReference($"Practitioner/{Practitioner.Id}"),
+                            new MedicationDispense.PerformerComponent()
+                            {
+                                Actor = new ResourceReference($"Practitioner/{Practitioner.Id}"),
+                            },
+                        },
+#if R5
```

Comment:
I see your point, but was trying to avoid maintaining these tests in 3 different projects and duplicating the Fixture as well. Do you still think it's better?

Reply:
I didn’t find a way to do it without duplicating all the common tests.
Not sure it’s better, unless I am missing something